low-level/files/opt/seagate/sspl/setup/resource_map/lr2/resource_map.py

```diff
@@ -95,3 +79,24 @@ def set_health_data(health_data: dict, status, description=None,
             "recommendation": recommendation,
             "specifics": specifics
         })
+
+
+class Manifest(metaclass=ABCMeta):
+    """Abstract class for all other manifest types."""
+
+    name = "manifest"
+
+    def __init__(self):
+        """Initialize manifest."""
+        logging_level = Conf.get(SSPL_CONF,
+            f"{SYSTEM_INFORMATION}>{LOG_LEVEL}", "INFO")
+        init_logging('node-manifest', logging_level)
+
+    @abstractmethod
+    def get_data(self, rpath):
+        """
+        Get technical information of fru in given manifest id.
```

Comment:
Why it cant be just get() ?

> Get manifest data for resource represented by resource path (rpath).

Reply:
https://github.com/Seagate/cortx-monitor/pull/442#discussion_r668087630
Please see above comment.